library/constant_time.c

```diff
@@ -741,6 +741,50 @@ int mbedtls_mpi_safe_cond_swap( mbedtls_mpi *X,
     return( ret );
 }
 
+/*
+ * Compare unsigned values in constant time
+ */
+unsigned mbedtls_mpi_core_lt_ct( const mbedtls_mpi_uint *X,
+                                 const mbedtls_mpi_uint *Y,
+                                 size_t len )
+{
+    size_t i;
+    /* The value of any of these variables is either 0 or 1 at all times. */
+    unsigned ret, cond, done;
+
+    ret = cond = done = 0;
+
+    for( i = len; i > 0; i-- )
```

Comment:
put the loop control variable into the loop, it's tidier and removes a declaration line, and supported even by MSVC!

```suggestion
    for( size_t i = len; i > 0; i-- )
```